App.js

```diff
@@ -3,18 +3,16 @@ import { StatusBar } from 'expo-status-bar';
 import { Text, View, Image } from 'react-native';
 import { NavigationContainer } from '@react-navigation/native';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
-import MdSearch from 'react-ionicons/lib/MdSearch'
-import MdNutrition from 'react-ionicons/lib/MdNutrition'
-import MdHome from 'react-ionicons/lib/MdHome'
+import { Ionicons } from '@expo/vector-icons';
```

Comment:
It's interesting that the react-ionicons weren't rendering on mobile with the IOSSearch / MdSearch format. Good call on using the vector-icons with expo!